rs/replay/src/player.rs

```diff
@@ -877,52 +878,13 @@ impl Player {
         &self,
         ingress_expiry: Time,
     ) -> Result<BlessedReplicaVersions, String> {
-        // Construct request.
-        let key = make_blessed_replica_versions_key();
-        let query = Query {
-            source: QuerySource::User {
-                user_id: UserId::from(PrincipalId::new_anonymous()),
-                ingress_expiry: ingress_expiry.as_nanos_since_unix_epoch(),
-                nonce: None,
-            },
-            receiver: REGISTRY_CANISTER_ID,
-            method_name: "get_value".to_string(),
-            method_payload: serialize_get_value_request(key.as_bytes().to_vec(), None)
-                .map_err(|err| format!("{}", err))?,
-        };
-
         self.certify_state_with_dummy_certification();
-
-        // Call get_value.
-        let reply: Vec<u8> = match self
-            .runtime
-            .block_on(self.query_handler.clone().oneshot((query, None)))
-            .unwrap()
-        {
-            Ok((Ok(WasmResult::Reply(reply)), _)) => reply,
-            garbage => {
-                return Err(format!(
-                    "Did not get reply from Registry get_value call: {:?}. key={}",
-                    garbage, key,
-                ));
-            }
-        };
-
-        // Parse (and possibly dechunkify) reply.
-        let record: Vec<u8> = self
-            .runtime
-            .block_on(deserialize_and_dechunkify_get_value_response(
-                reply,
-                &self.query_handler,
-                &key,
-            ))
-            .map_err(|err| format!("{:?}", err))?;
-        let record = deserialize_registry_value::<BlessedReplicaVersions>(Ok(Some(record)))
-            .map_err(|err| format!("{}", err))?
-            .expect("BlessedReplicaVersions does not exist");
-
-        // Done!
-        Ok(record)
```

Comment:
This was a near copy of some other code, and I had to modify this anyway, so I "took the opportunity" to factor out common code into a new stand-alone function: `registry_get_value` (defined further down).

Being a stand-alone function, `registry_get_value` is easier to test, because I do not need to figure out how to construct a valid Player.